graphx/src/main/scala/org/apache/spark/graphx/lib/StronglyConnectedComponents.scala

```diff
@@ -44,6 +44,11 @@ object StronglyConnectedComponents {
     // graph we are going to work with in our iterations
     var sccWorkGraph = graph.mapVertices { case (vid, _) => (vid, false) }.cache()
 
+    // helper variables to unpersist cached graphs
+    var prevSccGraph1 = sccGraph
+    var prevSccGraph2 = sccGraph
```

Comment:
I guess my logic is that it's easier to have one variable following this single variable throughout rather than two -- which of the two refers to which previous RDD? are they different? the effect of this as well is that you unpersist and the re-persist the RDD once during each loop.